designs/bee/src/band-tie.mjs

```diff
@@ -26,131 +27,125 @@ export const bandTie = {
     }
 
+    let bandTieWidth
+    if (options.crossBackTies || options.bandTieColours == 'one') {
+      bandTieWidth = absoluteOptions.bandTieWidth * 2
+    } else {
+      bandTieWidth = absoluteOptions.bandTieWidth
+    }
+    //let's begin
     points.topLeft = new Point(0, 0)
-    points.topRight = new Point(bandTieWidth * 2, points.topLeft.y)
+    points.topRight = new Point(bandTieWidth, points.topLeft.y)
     points.bottomLeft = new Point(points.topLeft.x, bandTieLength)
-    points.bottomRight = new Point(points.topRight.x, bandTieLength)
-
-    points.topMiddle = new Point(bandTieWidth, points.topLeft.y)
-    if (!options.crossBackTies && options.pointedTieEnds) points.topMiddle.y -= bandTieWidth
-
-    points.bottomMiddle = new Point(points.topMiddle.x, bandTieLength)
-
-    paths.seam = options.duoColorTies
-      ? new Path().move(points.bottomMiddle)
-      : new Path().move(points.bottomRight).line(points.topRight)
-    paths.seam.line(points.topMiddle).line(points.topLeft).line(points.bottomLeft).close()
+    points.bottomRight = new Point(points.topRight.x, points.bottomLeft.y)
+    points.topMid = new Point(points.topRight.x / 2, points.topLeft.y)
+    points.bottomMid = new Point(points.topMid.x, points.bottomLeft.y)
 
-    if (sa) paths.sa = paths.seam.offset(sa).attr('class', 'fabric sa')
-
-    /*
-     * Annotations
-     */
-
-    // Cut list
-    if (options.crossBackTies) store.cutlist.addCut({ cut: 1, from: 'fabric' })
-    else store.cutlist.addCut({ cut: 2, from: 'fabric' })
-
-    points.cofLeft = points.bottomLeft.shift(0, bandTieWidth * (1 / 8))
-    points.grainlineLeft = points.topLeft.translate(bandTieWidth * (1 / 8), bandTieLength * (3 / 4))
-    // Title
-    points.title = points.topLeft.translate(bandTieWidth * (1 / 8), bandTieLength * (1 / 4))
-    macro('title', {
-      at: points.title,
-      nr: 3,
-      title: options.crossBackTies ? 'band' : 'bandTie',
-      scale: 0.7,
-    })
-
-    // Foldline
-    if (options.duoColorTies) {
-      points.cofRight = points.bottomLeft.shift(0, bandTieWidth * (7 / 8))
-      points.grainlineRight = points.grainlineLeft.shift(0, bandTieWidth * (7 / 8))
+    if (options.bandTieEnds == 'straight' || options.crossBackTies) {
+      points.topPeak = points.topMid
     } else {
-      points.cofRight = points.bottomLeft.shift(0, bandTieWidth * (15 / 8))
-      points.grainlineRight = points.grainlineLeft.shift(0, bandTieWidth * (14 / 8))
-      if (complete)
+      if (options.bandTieColours == 'one') {
+        points.topPeak = points.topMid.shift(90, bandTieWidth / 2)
+      } else {
+        points.topPeak = points.topRight.shift(90, bandTieWidth)
+      }
+    }
+    //paths
+    paths.seam = new Path()
+      .move(points.bottomLeft)
+      .line(points.bottomRight)
+      .line(points.topRight)
+      .line(points.topPeak)
+      .line(points.topLeft)
+      .line(points.bottomLeft)
+      .close()
+
+    if (complete) {
```

Comment:
We don't do things like this anymore in v3. Which confirms my feeling that this is just outdated, code.
I'll stop my review here.